x-pack/plugins/apm/public/plugin.ts

```diff
@@ -73,6 +77,17 @@ export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
     pluginSetupDeps.home.environment.update({ apmUi: true });
     pluginSetupDeps.home.featureCatalogue.register(featureCatalogueEntry);
 
+    if (plugins.observability) {
+      const isDarkMode = core.uiSettings.get('theme:darkMode');
+      plugins.observability.dashboard.register({
+        appName: 'apm',
+        fetchData: async (params) => {
+          return fetchLandingPageData(params, { isDarkMode });
```

Comment:
I think it would be slightly better to make a separate function for getting the theme, and then passing that in:

```suggestion
      const isDarkMode = core.uiSettings.get('theme:darkMode');
      const theme = getTheme({ isDarkMode });
      plugins.observability.dashboard.register({
        appName: 'apm',
        fetchData: async (params) => {
          return fetchLandingPageData(params, { theme });
```